contrib/kafka-humio-gateway/main.go

```diff
@@ -0,0 +1,288 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/go-yaml/yaml"
+	"github.com/Shopify/sarama"
+)
+
+var (
+	verbose  = false
+	configFile string
+	defaultConsumerGroup = "velociraptor-consumer"
+)
+
+
+type TransportConfig struct {
+	Kafka struct {
+		Brokers []string
+		Topics []string
+		ConsumerGroup string `yaml:"consumer_group"`
+	}
+	Humio struct {
+		EndpointUrl string `yaml:"endpoint_url"`
+		IngestToken string `yaml:"ingest_token"`
+	}
+}
+
+func init() {
+	flag.BoolVar(&verbose, "verbose", false, "Sarama logging")
```

Comment:
"Sarama logging" is supposed to be the usage string, just having "Sarama logging" seems kind of ambiguous, perhaps a short explanation what does verbose enable would be better.